### DISCUSSION TITLE
Show coloured background where CD art would be if user is not using this feature

### DISCUSSION THREAD

**Discussion by takaji10** [TT-ReBORN/Georgia-ReBORN#32] [Q&A] [answered]
What do you think about filling this blank space to the right of the album art with the same blue colour that is generated in the details pane when CD art is not being used?

![image](https://user-images.githubusercontent.com/1534470/127836192-25b01486-c12f-4e32-87c3-5157f077b01c.png)

**Comment by TT-ReBORN** [comment]
Great idea!

I will implement this as an option and enable this by default.
What should I name this option? Show background when no disc art?

**Reply by takaji10** [reply]
I think you should just make it the default state when there's no CD art being shown. Is there a reason someone would not want to show it?

**Reply by TT-ReBORN** [reply]
Aesthetics, different design taste ( for someone it might look boxed, no space to breath ) and freedom of design choice.

**Reply by takaji10** [reply]
Then in that case I'd use "Show colour background when disc art disabled"

**Comment by TT-ReBORN** [comment, marked as answer]
Done!
